obspy/io/nordic/core.py

```diff
@@ -466,6 +474,50 @@ def _read_uncertainty(tagged_lines, event):
     return event
 
 
+def _read_highaccuracy(tagged_lines, event):
+    """
+    Read high accuracy origin line.
+    """
+    if 'H' not in tagged_lines.keys():
+        return event
+    # In principle there shouldn't be more than one high precision line
+    line = tagged_lines['H'][0][0]
+    try:
+        dt = {'Y':  int(line[1:5]),  'MO': int(line[6:8]),
+              'D':  int(line[8:10]),  'H': int(line[11:13]),
+              'MI': int(line[13:15]), 'S': float(line[16:23])}
+    except ValueError:
+        pass
+    try:
+        ev_time = UTCDateTime(dt['Y'], dt['MO'], dt['D'],
+                              dt['H'], dt['MI'], 0, 0) + dt['S']
+        if abs(event.origins[0].time - ev_time) < 0.1:
+            event.origins[0].time = ev_time
+        else:
+            print('High accuracy time differs from normal time by >0.1s')
+    except ValueError:
+        pass
+    try:
+        values = {'latitude': float(line[23:32]),
+                  'longitude': float(line[33:43]),
+                  'depth': float(line[44:52]), 'rms': float(line[53:59])}
+    except ValueError:
+        pass
+    if values['latitude']:
```

Comment:
These should all compare to `None`, just in case you have an event at 0, 0, 0.

Reply:
Now use your float_conv routines and compare to None